File: svt-projekat2024/src/main/java/rs/ac/uns/ftn/svt/model/Comment.java
package rs.ac.uns.ftn.svt.model;

import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDateTime;
import java.util.List;

@Data
@Entity
public class Comment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String text;

    private LocalDateTime createdAt;

    @OneToMany(mappedBy = "comment")
    private List<Review> reviews;

}

File: kvt-projekat2024/src/app/models/Comment.ts
import { Review } from "./Review";

export class Comment {
  id: number;
  text: string;
  createdAt: Date;
  reviews: Review[];

  constructor(id: number, text: string, createdAt: Date, reviews: Review[]) {
    this.id = id;
    this.text = text;
    this.createdAt = createdAt;
    this.reviews = reviews;
  }
}
